src/stats/rawsender.h

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2017-2023 Vincent Thiery
+// Copyright (c) 2024 The Dash Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_STATS_RAWSENDER_H
+#define BITCOIN_STATS_RAWSENDER_H
+
+#include <compat.h>
+#include <sync.h>
+#include <threadinterrupt.h>
+
+#include <deque>
+#include <memory>
+#include <optional>
+#include <string>
+#include <vector>
+
+class Sock;
+
+struct RawMessage : public std::vector<uint8_t>
```

Comment:
`RawSender` is not a good name for this class, because it's just std::vector and has nothing to do with 'RawSender'.

Move this one to client.h and maybe rename it to 'RawStat' or `StatRawMessage` or something similar.

Reply:
`RawMessage` was made explicitly for `RawSender` ([source](https://github.com/dashpay/dash/blob/ad63686c95f58873c6fffd524bbaa2ae3366ef9b/src/stats/rawsender.h#L66-L74), [source](https://github.com/dashpay/dash/blob/ad63686c95f58873c6fffd524bbaa2ae3366ef9b/src/stats/rawsender.h#L87)), please refer to discussion on `RawMessage` ([source](https://github.com/dashpay/dash/pull/6267#discussion_r1757355058)).